apps/trading/components/accounts-container/accounts-container.tsx

```diff
@@ -75,3 +85,31 @@ export const AccountsContainer = ({
     </div>
   );
 };
+
+type Store = {
```

Comment:
Store shape is the same as `State` from libs\datagrid\src\lib\use-datagrid-events it will be better if we can import and reuse it